### ISSUE TITLE
Not Working When Combined with Custom Attachment Location Plugin

### ISSUE THREAD

**Issue by Drakemoor** [xRyul/obsidian-image-converter#26]
When Image Converter is used in combination with "[Custom Attachment location](https://github.com/RainCat1998/obsidian-custom-attachment-location#obsidian-custom-attachment-location)" plugin.
It does nothing. 

It doesn't matter if I paste the image or if I drag and drop it. It just doesn't do anything. The image that ends into the vault is the same file format as the original image and has the same name. 

There is no error in the console. 

Thanks for the great plugin!

**Comment by xRyul** [comment]
Hey, many thanks for checking it out.

Could you show me screenshot of settings for both plugins please?

I have just installed Custom Attachemnt location plugin and used default setting for both:
- it processed the file just fine
-  it also renamed the file on paste and on drop and it also used the save location as defined by "Custom Attachemnt location"

**Comment by Drakemoor** [comment]
Here there are:
![Custom Attachment Location Plugin Settings](https://github.com/xRyul/obsidian-image-converter/assets/93224920/b29cecb3-02f9-4168-a68f-f499c36c6217)

![Image Converter Plugin Settings](https://github.com/xRyul/obsidian-image-converter/assets/93224920/095e4e40-efc9-453c-9672-d1b2c4b6f931)

Let me know if I can do anything else to pinpoint the issue. 

Thank you for the great plugin!

**Comment by xRyul** [comment]
Hey, it's been a while and your workflow might have changed, however latest release adds many custom options (variables) for custom file naming and folder creation. You can find some of the examples [here](https://github.com/xRyul/obsidian-image-converter/blob/master/Examples/Variables%20Reference%20Guide.md) . Let me know if this helps 🚀

![image](https://github.com/user-attachments/assets/edff313e-1ec5-4aa8-b3c7-b99019ccbe8c)

![image](https://github.com/user-attachments/assets/d0e454b5-ef61-4253-8a9f-3bc46624f705)

![image](https://github.com/user-attachments/assets/f4afe2c6-7d36-4ee6-a593-139950711f42)

**Comment by Drakemoor** [comment]
Thank you!

I've got the notification for the changes. Everything looks super well organized, and useful. I've made some adjustments to my workflow, and presently, the plugin works flawlessly.  

I love the new 'Toggle Image Conversion' command, especially because I can add it as a button. 

I would continue to test and if I find any issues I would report accordingly. 

Again, thank you or the great plugin. ❤🙇‍♂️
